### DISCUSSION TITLE
Looking for input prompt that includes a desired number of words/sentences/paragraphs in the response.

### DISCUSSION THREAD

**Discussion by Coffeeworld** [LostRuins/koboldcpp#1212] [Q&A] [answered]
I'm a novice to AI/LLM in general and am searching for a function that would allow me to include a desired number of words, sentences, or paragraphs in the model's response to a prompt - something that could be defined for each prompt. I think I've seen the ability to set the number of tokens in a response during setup, but not on a per-prompt basis.

**Comment by LostRuins** [comment]
You can't really force a specific number of specific words. You can limit the number of total generated tokens, but the choice of word is up to the model

**Comment by jabberjabberjabber** [comment, marked as answer]
Command-r can do this, but most models cannot.

* https://docs.cohere.com/docs/crafting-effective-prompts#length-control
